data/ga/ACCESSCONTROL-DEPARTMENTROLES/departmentroles.json

```diff
@@ -49,6 +49,18 @@
 		{
 			"departmentcode": "ENGS",
 			"rolecode": "BPA_COL_INITIATOR"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_AM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_DGM"
+		},
+		{
+			"departmentcode": "ESTD",
+			"rolecode": "ESTD_VERIFIER_GM"
```

Comment:
Also, as discussed in last call,
we do not want to mention any designation details in the roles.

So, please remove the AM, DGM and GM keywords from the Roles.
And as per your level, you can add L1, L2, L3.

For example: 
{
		"departmentcode": "ESTD",
		"rolecode": "ESTD_VERIFIER_L1"
}